libpod/container_inspect.go

```diff
@@ -0,0 +1,70 @@
+package libpod
+
+import (
+	"github.com/projectatomic/libpod/libpod/driver"
+	"github.com/sirupsen/logrus"
+)
+
+func (r *Runtime) getContainerInspectData(ctr *Container, size bool, driverData *driver.Data) (*ContainerInspectData, error) {
+	config := ctr.config
+	runtimeInfo := ctr.state
+	spec := ctr.config.Spec
+
+	args := config.Spec.Process.Args
+	var path string
+	if len(args) > 0 {
+		path = args[0]
+	}
+	if len(args) > 1 {
+		args = args[1:]
+	}
+
+	data := &ContainerInspectData{
+		ID:      config.ID,
+		Created: config.CreatedTime,
+		Path:    path,
+		Args:    args,
+		State: &CtrState{
+			OciVersion: spec.Version,
+			Status:     runtimeInfo.State.String(),
+			Running:    runtimeInfo.State == ContainerStateRunning,
+			Paused:     runtimeInfo.State == ContainerStatePaused,
+			OOMKilled:  runtimeInfo.OOMKilled,
+			Dead:       runtimeInfo.State.String() == "bad state",
```

Comment:
I think this is probably ContainerStateStopped, not bad state?

Reply:
I tested with a stopped docker container, and Dead is false. So I am assuming it is Dead if it is a bad state.